app/firebase/functions/test/__tests__/setup.ts

```diff
@@ -58,12 +59,27 @@ export const mochaHooks = (): Mocha.RootHookObject => {
         if (fs.existsSync(TEST_USERS_FILE_PATH)) {
           const fileContents = fs.readFileSync(TEST_USERS_FILE_PATH, 'utf8');
           appUsers = JSON.parse(fileContents);
-
-          await Promise.all(
-            appUsers.map(async (appUser) => {
-              testUsers.set(appUser.userId, appUser);
-            })
-          );
+          if (
+            checkOutdatedTwitterTokens(appUsers) ||
```

Comment:
In principle, even if expired, we should be able to use the refresh token (automatically) and update the access token as part of the test. At the end the test.users.json file should be rewritten with the new access token and refresh token.

For some reason this is not working, suggesting the refresh token stored in test.users.json is not the valud one.